### DISCUSSION TITLE
Where to buy doorbell plates with 25 mm hole?

### DISCUSSION THREAD

**Discussion by BerhardOhneN** [frickelzeugs/FingerprintDoorbell#37] [Q&A] [answered]
Hi Frickelzeugs,

congratulations for the great project! I'm currently working on one for my home.
Can you tell me whre you purchased your doorbell plate? I'm searching for a plate with 25 mm hole and can't find one on the common marketplaces in Germany (Metzler, etc.).

Thanks a lot!
Bernhard

**Comment by frickelzeugs** [comment, marked as answer]
Hi, mine was purchased from Metzler a few years ago, but with a 20mm hole and 19mm bell button. When I converted the old bell plate to the fingerprint doorbell, I asked a local metalworker if he could drill the 20mm hole to 25mm.

**Reply by baetzmr** [reply]
Hi. Metzler Doorbell with 25mm LED Button here. You have to compare a few Metzler Variants and search for 25mm Models
![image](https://user-images.githubusercontent.com/81546284/201998245-a3e9ddbb-bc48-4923-a730-d80fe0baffd9.png)

![image](https://user-images.githubusercontent.com/81546284/201998302-b1760349-4d7b-49b5-992f-ca4f3447f2dd.jpeg)

**Reply by BerhardOhneN** [reply]
Thanks a lot for your suggestions!
